README.md

```diff
@@ -55,7 +55,7 @@ From there, you'll need to add a line to your form:
 
 edit.erb:
 ```html
-<form action="/models/:id" method="post">
```

Comment:
@genericlady it looks like we should really change this to `"/models/<%=@model.id%>"`. Otherwise when I tested it I got the action path being set to something like `"/models/2/2"`. If you update this and just push to the branch, the PR will auto update. 
If you at_metntion me when you do this, we should be able to merge it in.
Thanks for catching this.